### DISCUSSION TITLE
Timestamps accuracy?

### DISCUSSION THREAD

**Discussion by RaulKite** [openai/whisper#647] [Q&A] [answered]
Hi @jongwook, 
Does this new large model improve timestamps accuracy?

Have you any plans to give token timestamps in any future release?

Any way or workaround to have them ?

Thank you for all

Your work is fantastic 😊

**Comment by jongwook** [comment, marked as answer]
Thanks :) We haven't compared timestamp accuracy between the two models, but I expect the new one would be marginally better. While I can't commit to new feature releases at this time, you can try community implementations like those in #435.
